passes/techmap/abc9_ops.cc

```diff
@@ -1202,7 +1202,7 @@ void reintegrate(RTLIL::Module *module, bool dff_mode)
 	}
 
 	std::vector<Cell*> boxes;
-	for (auto cell : module->cells().to_vector()) {
+	for (auto cell : module->cells()) {
```

Comment:
Note that `to_vector` creates a copy of the cell list, necessary because the body of the `for` loop is modifying this very same list.